controller/node_controller.go

```diff
@@ -985,7 +984,7 @@ func (nc *NodeController) syncInstanceManagers(node *longhorn.Node) error {
 				}
 			}
 			if cleanupRequired {
-				logrus.Infof("Cleaning up the redundant instance manager %v when there is no running/starting instance", im.Name)
+				logrus.Infof("Cleaning up the redundant instance manager %v when there is no running/starting instance: %+v", im.Name, im.Status.Instances)
```

Comment:
`im.Status.Instances` could be a lot, so it would overwhelm logs. What exact things you would like to look into like instances with some specific state? probably intentionally log them instead and use debug level log.

Reply:
That's a good point. Thanks for it.
I will try to get the fields I need!